skribenten-web/frontend/src/pages/Brevvelger/SelectedTemplate.tsx

```diff
@@ -156,21 +165,38 @@ function LetterTemplateHeading({ letterTemplate }: { letterTemplate: LetterMetad
           margin-top: var(--a-spacing-2);
         `}
       >
-        <div
-          aria-hidden
-          css={css`
-            width: 8px;
-            height: 8px;
-            border-radius: 50%;
-            background: var(--a-green-500);
-          `}
-        />
-        <BodyShort size="small">REDIGERBAR {letterTemplate.isEblankett ? "BLANKETT" : "MAL"}</BodyShort>
+        <LetterTemplateTags letterTemplate={letterTemplate} />
       </div>
     </div>
   );
 }
 
+function LetterTemplateTags({ letterTemplate }: { letterTemplate: LetterMetadata }) {
+  if (letterTemplate.isEblankett) {
+    return <></>;
+  }
+
+  return (
+    <div>
+      {letterTemplate.brevsystem === BrevSystem.Brevbaker && (
```

Comment:
Bruke en exhaustive switch her? Så får man kompileringsfeil om det skulle komme flere enumerasjoner (ikke at det er spesielt sannsynlig i akkurat dette tilfellet da)

Reply:
Sikkert lurt 👍 Trengte å endre litt på tsconfig for at den faktisk skulle rapportere feil